gateway/tests/harness.rs

```diff
@@ -600,8 +646,8 @@ impl Harness {
                 .unwrap(),
             )?;
             rt.verify();
-            let st: State = rt.get_state();
-            assert_eq!(st.applied_bottomup_nonce, msg_nonce);
+            // let st: State = rt.get_state();
```

Comment:
Uncomment? We need to be sure that the nonce logic is done correctly. If it fails we need to know why.